suite/tests/linux/clone.c

```diff
@@ -298,7 +297,6 @@ make_clone3_syscall(void *clone_args, ulong clone_args_size, void (*fcn)(void))
     return result;
 }
 
-#ifdef SYS_clone3
```

Comment:
Hmmm. The removal of the #ifdef breaks the android build. Replace with #ifndef ANDROID?
[here and at the call site]
I don't know enough about android support to be confident in what the right fix is though.